zeppelin-zengine/src/main/java/org/apache/zeppelin/notebook/repo/zeppelinhub/websocket/ZeppelinClient.java

```diff
@@ -65,6 +69,12 @@
   private Authentication authModule;
   private static final int MIN = 60;
 
+  private static final Set<String> actionable = new  HashSet<String>(Arrays.asList(
```

Comment:
i tried to keep it small, updated ops in 4616a334140d1039db1a3b37bf5434e129825e6e